openlibrary/accounts/model.py

```diff
@@ -603,7 +602,8 @@ def s3auth(cls, access_key, secret_key):
             return response.json()
         except requests.HTTPError as e:
             return {'error': e.response.text, 'code': e.response.status_code}
-        except simplejson.errors.JSONDecodeError as e:
+        except ValueError as e:
+            # we cannot catch json.JSONDecodeError here yet because requests will import simplejson as json first.
```

Comment:
I get `json.decoder.JSONDecodeError` locally.

% ___python3___
```
Python 3.9.1 (default, Jan  8 2021, 12:11:08)
[Clang 12.0.0 (clang-1200.0.32.28)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import requests
>>> r = requests.get('http://httpbin.org/status/200')
>>> r.status_code
200
>>> r.json()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/homebrew/lib/python3.9/site-packages/requests/models.py", line 900, in json
    return complexjson.loads(self.text, **kwargs)
  File "/opt/homebrew/Cellar/python@3.9/3.9.1_6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/opt/homebrew/Cellar/python@3.9/3.9.1_6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/opt/homebrew/Cellar/python@3.9/3.9.1_6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

Reply:
Oh...  Now I see https://www.peterbe.com/plog/jsondecodeerror-in-requests.get.json-python-2-and-3

Let's use this style
```python
try:
    from simplejson.errors import JSONDecodeError
except ImportError:
    from json.decoder import JSONDecodeError
```
Then we can streamline when we finally remove simplejson from our requirements*.txt files.